release/release.sh

```diff
@@ -23,20 +23,21 @@ function new_release {
 
 
 function bump_version {
-  echo "Bumping main.go version ..."
-  sed -i -E "s/(pulp-operator version:) .*/\1 ${PULP_OPERATOR_DEV_VERSION}\")/g" main.go
-
   echo "Bumping Makefile operator's version ..."
-  sed -i -E "s/^(VERSION \?=) .*/\1 ${PULP_OPERATOR_DEV_VERSION}/g" Makefile
+  sed -i -E "s/^(VERSION \?=) .*/\1 ${PULP_OPERATOR_RELEASE_VERSION}/g" Makefile
 
   echo "Bumping the tag used in bundle-upgrade pipeline  ..."
   sed -i -E "s/(ref:) ${PULP_OPERATOR_REPLACE_VERSION}/\1 ${PULP_OPERATOR_RELEASE_VERSION}/g" .github/workflows/ci.yml
 
   echo "Updating the manifests with the changes ..."
   make generate manifests bundle
 
+  echo "Updating containerImage annotation"
+  CSV_FILE=manifests/pulp-operator.clusterserviceversion.yaml
+  sed -i "s#containerImage: quay.io/pulp/pulp-operator:devel#containerImage: quay.io/pulp/pulp-operator:v${PULP_OPERATOR_RELEASE_VERSION}#g" $CSV_FILE
```

Comment:
There is no reason for CSV not to be as similar as possible between pulp-operator release, k8s community release, and openshift community release. So, this should be harmless.